checker/src/org/checkerframework/checker/index/samelen/SameLenAnnotatedTypeFactory.java

```diff
@@ -214,4 +214,8 @@ public AnnotationMirror createSameLen(String... val) {
         builder.setValue("value", val);
         return builder.build();
     }
+
+    public AnnotationMirror createSameLenUnknown() {
```

Comment:
This method isn't creating anything, so the method name is confusing.  Either change the name or delete the method and make the field public.